plugin/src/main/scala/com/softwaremill/clippy/AdviceLoader.scala

```diff
@@ -27,7 +27,7 @@ class AdviceLoader(global: Global, url: String, localStoreDir: File, projectAdvi
     TryWith(url.openStream())(inputStreamToClippy(_).advices) match {
       case Success(advices) => advices
       case Failure(_) =>
-        global.warning(s"Cannot load advice from ${url.getPath} : Ignoring.")
+        global.inform(s"Cannot load advice from ${url.getPath} : Ignoring.")
```

Comment:
I dropped this to inform to prevent breaking builds for users who have scalac treat warnings as errors.